lib/gitlib_test.sh

```diff
@@ -8,6 +23,9 @@ source $TOOL_LIB_PATH/releaselib.sh
 
 readonly TESTDATA="$( cd "$(dirname "$0")" && pwd )/testdata"
 
+set -e
```

Comment:
Prefer:
```
set -o errexit
set -o nounset
set -o pipefail
```

and set options immediately after file header.

Reply:
Yeah, we can do that when all the things we source ( == things we want to test and they depend on [`gitlib.sh`, `common.sh`, `releaselib.sh`]) don't run code that relies on undefined variables and doesn't ignore errors and all that jazz. When can we do that? When we have tests in place. I guess you see where I am going here ...